odoo/addons/openupgrade_records/models/comparison_config.py

```diff
@@ -81,9 +80,10 @@ def install_modules(self):
 
         modules = []
         for module_id in remote_module_ids:
-            name = remote_module_obj.read(module_id, ["name"])
-            name = apriori.renamed_modules.get(name, name)
-            modules.append(name)
+            mod = remote_module_obj.read([module_id], ["name"])[0]
```

Comment:
I think the `browse` fails because the models are not from `env['model']` but from `connection.get_model('model')`